halo2_proofs/src/arithmetic.rs

```diff
@@ -139,6 +139,12 @@ pub fn small_multiexp<C: CurveAffine>(coeffs: &[C::Scalar], bases: &[C]) -> C::C
     acc
 }
 
+#[cfg(feature = "cusnark")]
+/// Performs a GPU accelerated multi-exponentiation operation with cuSnark
+pub fn cusnark_multiexp<C: CurveAffine>(coeffs: &[C::Scalar], bases: &[C]) -> C::Curve {
+    todo!();
+}
+
```

Comment:
So is this going to call our internal best_multiexp, or are we migrating that here (is there no longer a need for the best_multiexp function in our cusnark rust bindings?)

Reply:
Good point. I think replicating `best_multiexp` is not a good design, since by its name `best_multiexp` is supposed to choose from different MSM implementations automatically.

You are right, we can call `cusnark::halo2::msm` directly from `cusnark_multiexp` without the `cusnark::halo2:msm` interface.